dotcom-rendering/cypress/integration/parallel-3/liveblog.interactivity.spec.js

```diff
@@ -201,7 +201,7 @@ describe('Liveblogs', function () {
 			cy.get('#46d194c9-ea50-4cd5-af8b-a51e8b15c65e').should(
 				'not.be.visible',
 			);
-			cy.get('[data-gu-marker=top-of-blog]').scrollIntoView();
+			cy.get('[id=top-of-blog]').scrollIntoView();
```

Comment:
Using an id here s we can use this element as an anchor tag

Reply:
Not sure if it makes sense as I am uncertain of what `data-gu-marker` is, but if it's a contract between us and others, maybe we could have both `id` and `data-gu-marker` that serve different purposes?

Reply:
The data attribute is an attempt to move away from using `id`s everywhere but in this case, where we explicitly want an anchor tag, it made more sense to go with that.

I think the data- pattern is emerging. It's nice because you have a more space for description, you can specify both a key and a value, but I don't think there are any hard contracts yet. One contract we _do_ have is to only use classnames for styling. So maybe data attributes can be thought of as alternatives for where classnames used to be used as selectors.